cluster/gce/configure.sh

```diff
@@ -155,6 +155,8 @@ if [ -n "${network_policy_provider}" ] && [ "${network_policy_provider}" != "non
 fi
 log_level="${CONTAINERD_LOG_LEVEL:-"info"}"
 cat > ${config_path} <<EOF
+disabled_plugins = ["restart"]
```

Comment:
nit.. can we explain or link to the reason for disabling restart here... for example:
```# containerd provides it's own restart manager```

Reply:
Will do.

Reply:
Done